src/zenml/integrations/kubernetes/flavors/kubernetes_orchestrator_flavor.py

```diff
@@ -147,6 +147,15 @@ def docs_url(self) -> Optional[str]:
         """
         return self.generate_default_docs_url()
 
+    @property
+    def sdk_docs_url(self) -> Optional[str]:
+        """A url to point at SDK docs explaining this flavor.
+
+        Returns:
+            A flavor SDK docs url.
+        """
+        return self.generate_default_sdk_docs_url()
```

Comment:
After seeing what seems like 100 copies of this, it seems very likely that this could be the base class implementation that checks whether a class is "builtin" or not and returns the default docs if it is.